### PR TITLE
Arc interpolation fixing and added support for circle type

### PR DESCRIPTION
**Pull request by arunverma87**
PR related to #228

### CODE REVIEW

File: io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java [hunk 1]
```diff
@@ -160,10 +162,11 @@ public Object createGeometry(Instance instance, int srsDimension, IOProvider rea
 						// GeometryProperty<LinearRing> instance
 						GeometryProperty<LinearRing> ring = (GeometryProperty<LinearRing>) ((Instance) value)
 								.getValue();
-						outerRing.add(ring.getGeometry());
+						outer = ring.getGeometry();
```
Comment (stempler) [inline io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java]: What is the reason for this change?

File: util/plugins/eu.esdihumboldt.util.test/src/eu/esdihumboldt/util/geometry/interpolation/CircleInterpolationTest.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	}
+
+//0.01,3.2 3.33,3.33 0.01,-3.2
+	/**
+	 * test algorithm
+	 */
+	@Test
+	public void testInterpolation() {
+		System.out.println("-- Test-" + testIndex + " begin --");
+		if (skipTest) {
```
Comment (stempler) [inline util/plugins/eu.esdihumboldt.util.test/src/eu/esdihumboldt/util/geometry/interpolation/CircleInterpolationTest.java]: What is the purpose of this condition?

The danger I see here that looking at the test results you will never know that a test was actually skipped, because it will be listed as passed test.

Why should tests be skipped?
If there is a use case for that it might be wiser to use JUnit `assume*` methods (see [here](http://stackoverflow.com/questions/1689242/conditionally-ignoring-tests-in-junit-4) for an example).

File: io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java (hunk 1 above)
Comment (arunverma87) [inline io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java]: Before I was adding each value to list but I think we need a last value, while iterating each value.

Comment (stempler) [inline io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java]: Did you encounter a specific problem with this? Because generally I think there should only be one outer ring, and in this case the behavior of both approaches should actually be the same. So did you encounter a case where there were multiple outer rings found here?

File: util/plugins/eu.esdihumboldt.util.test/src/eu/esdihumboldt/util/geometry/interpolation/CircleInterpolationTest.java (hunk 2 above)
Comment (arunverma87) [inline util/plugins/eu.esdihumboldt.util.test/src/eu/esdihumboldt/util/geometry/interpolation/CircleInterpolationTest.java]: I created parameterized tests with around 16 test cases, and this one is only for my convenience, if I want to run only single test while debugging.

File: io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java (hunk 1 above)
Comment (arunverma87) [inline io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java]: No I couldn't encounter any issue with that, but i changed it again, just to ensure no problem should be generated because of interpolation any more.

Comment (stempler) [inline io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java]: If you changed it again it did not make it into the PR
